openshift/openshift-copies.go

```diff
@@ -210,13 +209,13 @@ func (config *directClientConfig) ClientConfig() (*restConfig, error) {
 		if err != nil {
 			return nil, err
 		}
-		mergo.MergeWithOverwrite(clientConfig, userAuthPartialConfig)
+		_ = mergo.MergeWithOverwrite(clientConfig, userAuthPartialConfig)
```

Comment:
I think we should check these errors. I'm not familiar with the `openshift` transport but it doesn't seem right to silently ignore.